ipn/ipnlocal/local.go

```diff
@@ -5012,7 +5005,9 @@ func (b *LocalBackend) ResetForClientDisconnect() {
 	b.setNetMapLocked(nil)
 	b.pm.Reset()
 	if b.currentUser != nil {
-		b.currentUser.Close()
+		if c, ok := b.currentUser.(io.Closer); ok {
```

Comment:
either define your own interface like `ActorCloser` or add a `Close` func to the `Actor` interface with docs that its typically a no-op but implementations can choose to define one if they need it.

Reply:
I went with the `ActorCloser` interface, thanks!